README.md

```diff
@@ -11,21 +11,105 @@ This project is licensed under the Apache License 2.0
 
 ## Usage
 
-csalt accepts the following parameters:
-1. Device and Groups. A list of Devices or group names to translate separated by a space
-	- Devices must be in the format of groupname:devicename
+
+```
+usage: csalt [-s] [--server SERVER] [--user USER]
+                        [--test] [--live] [-t] [-d]
+                        DEVICEINFO COMMANDS
+
+positional arguments:
+  DEVICEINFO            Comma separated list of device/group names to translate salt ids on (See below for more details)
+  COMMANDS              Salt commands to run on these devices e.g. `state.apply test=True`, this can be 1 or multiple commands
+
+optional arguments:
+  -h, --help            Show this help message and exit
+  -s --show             Print salt ids for device names, this will override
+  --server SERVER
+                        Use server configuration for the specified server alias in cacophony-user.yaml
+                        servers:
+                          <SERVER ALIAS>:
+                            url: http://127.0.0.1:1080/
+                            salt-prefix: test
+                            user-name: admin_test
+  --user USER           Use this user name to authenticate with, this will be saved to cacophony-user.yaml
+  --test                Connect to test api server https://api-test.cacophony.org.nz/
+  --live                Connect to live api server https://api.cacophony.org.nz/
+  -t --test-prefix      Append test to salt ids e.g. pi-test-XXX
+  -d --debug            Enable debug mode with extra logging
+```
+
+DEVICEINFO:
+1. Device and Groups. A list of Devices or group names to translate separated by a comma
+	- Devices can be in the format of groupname:devicename, or :devicename (which will match any group)
 	- Groups will be translated into all devices in this group
-2. Salt commands to run e.g. `state.apply test=True`, this can be 1 or multiple commands
 
 If only 1 parameter is supplied this will run directly on salt
 
+Once a user has been authenticated a temporary token will be saved to /home/user/.cacophony-token
```

Comment:
Can this and some examples also go into the `csalt -h` output?

Reply:
Yep done.